src/main/resources/application-jrd-router-orchestartion.yaml

```diff
@@ -52,8 +52,8 @@ route:
     truncate-sql:
       sql:TRUNCATE TABLE judicial_office_authorisation RESTART IDENTITY?dataSource=dataSource
     insert-sql:
-      sql:INSERT INTO judicial_office_authorisation (judicial_office_auth_id,per_id,jurisdiction,ticket_id,start_date,end_date,created_date,last_updated,lower_level,personal_code,object_id)
-      VALUES(:#judicial_office_auth_id,:#per_id,:#jurisdiction,:#ticket_id,:#start_date,:#end_date,NOW() AT TIME ZONE 'utc',NOW() AT TIME ZONE 'utc',:#lower_level,:#personal_code, :#object_id)?batch=true&dataSource=dataSource
+      sql:INSERT INTO judicial_office_authorisation (judicial_office_auth_id,per_id,jurisdiction,ticket_id,start_date,end_date,created_date,last_updated,lower_level,personal_code,object_id,mrd_created_time,mrd_updated_time,mrd_deleted_time)
+      VALUES(:#judicial_office_auth_id,:#per_id,:#jurisdiction,:#ticket_id,:#start_date,:#end_date,NOW() AT TIME ZONE 'utc',NOW() AT TIME ZONE 'utc',:#lower_level,:#personal_code, :#object_id ,TO_TIMESTAMP(NULLIF(:#mrd_created_time,''), 'YYYY-MM-DD HH24:MI:SS') ,TO_TIMESTAMP(NULLIF(:#mrd_updated_time,''), 'YYYY-MM-DD HH24:MI:SS') ,TO_TIMESTAMP(NULLIF(:#mrd_deleted_time,''), 'YYYY-MM-DD HH24:MI:SS') )?batch=true&dataSource=dataSource
```

Comment:
I think you need not do any conversion to timestamp like 
`TO_TIMESTAMP(NULLIF(:#mrd_created_time,'')`
. you can just keep it simple like (:#mrd_created_time) and do the conversion in java side like https://github.com/hmcts/rd-judicial-data-load/blob/master/src/main/java/uk/gov/hmcts/reform/juddata/camel/mapper/JudicialRoleTypeRowMapper.java#L21